packages/cli/src/active-executions.ts

```diff
@@ -174,6 +174,22 @@ export class ActiveExecutions {
 		this.logger.debug('Execution finalized', { executionId });
 	}
 
+	/** Resolve the response promise in an execution. */
+	resolveExecutionResponsePromise(executionId: string) {
```

Comment:
could we use the existing `resolveResponsePromise` method? I'm just wondering if we can avoid the duplication here.